### PR TITLE
Images are now shown properly in landscape and portrait mode

### LINKED ISSUE #1144: Image Not Viewing Properly while toggling between Landscape and Portrait
**Actual Behaviour**
2 images are shown instead of one while toggling between landscape and portrait and image is also not properly positioned.

**Expected Behaviour**

It should be properly positioned and viewed while switching between the landscape and portrait only that specific image should be viewed in that mode instead of two with distorted positions.

**Steps to reproduce it**
Go to a folder open image now change mode to landscape and then to portrait.

**LogCat for the issue**

09-15 02:15:35.993 7870-7870/org.fossasia.phimpme I/PhotoViewAttacher: ImageView no longer exists. You should not use this PhotoViewAttacher any more.

**Screenshots of the issue**

screenshot_2017-09-15-02-10-13

**Would you like to work on the issue?**
Yes!

@mohitmanuja  @anantprsd5  kindly review the issue!

### PR DESCRIPTION
**Pull request by batbrain7**
Fix #1144 

Changes: 
The problem that was shown in the logcat as  : " org.fossasia.phimpme I/PhotoViewAttacher: ImageView no longer exists. You should not use this PhotoViewAttacher any more."

This was because of the old version of the photoview which was not able to apprehend the null image. Now, I added a new better version of this photoview, which does not show this error. Also in the manifest the configchanges when set to orientation and the screenSize do not allow the programmable code to change the configuration. therefore fixed that too.
Now, the images on changing the orientation do are shown properly with appropriate resolution and no 
overlapping is seen in them as was before in the issue.

Screenshots for the change: 
screenshot_20171228-171826
screenshot_20171228-171850

### CODE REVIEW

**Comment by batbrain7** [comment]
@saarthakchats I had squashed earlier commits, but I think the difference is huge therefore 4 commits are suffice to show it

File: gradle.properties [hunk 1]
```diff
@@ -1,27 +0,0 @@
-# Project-wide Gradle settings.
```
Comment (abishekvashok) [inline gradle.properties]: why was this deleted?

Comment (batbrain7) [reply gradle.properties]: I think when I added added new changes to git I added some unnecessary files too, so while resetting the added changes I think it got deleted.
Fixing this right away

**Comment by abishekvashok** [comment]
@anantprsd5 can you take a look pls

**Comment by mohitmanuja** [comment]
@batbrain7 Please push the actual changes there is a lot of unnecessary changes. Please push the changes which can be clearly visible.

**Comment by batbrain7** [comment]
@mohitmanuja these are the actual changes
